src/tree.component.ts

```diff
@@ -83,4 +86,16 @@ export class TreeComponent implements OnInit, OnChanges {
       this.nodeCollapsed.emit(e);
     });
   }
+
+  public getController(): TreeController {
+    return this.rootComponent.controller;
+  }
+
+  public getChildControllerById(id: number | string): TreeController {
```

Comment:
@lovery we can just rename this method to `getTreeControllerByNodeId` and that's it